muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsJsPlugin_en.properties

```diff
@@ -41,7 +41,10 @@ plugin.records.summary.notifications.title = Notifications
 plugin.records.yo.title = Your matricular examinaition
 plugin.records.yo.abiStatus.title = ABI-status
 plugin.records.yo.abiStatus.content.finished = Your ABI-status is does not lack courses. You can now apply for matricular examination.
-plugin.records.yo.button.signUp = Apply for matricular examination
+plugin.records.yo.button.signUp.active = Apply for matricular examination
+plugin.records.yo.button.signUp.active.title = Application ends {0}
+plugin.records.yo.button.signUp.inactive.title = Today is {0}
+plugin.records.yo.button.signUp.inactive = Applying starts at {0}
 plugin.records.yo.participation.title = Participating in matricular examination
```

Comment:
Not part of this PR but should be changed. Suggested change: Participating in matriculation examination